test/controllers/publishers_controller_test.rb

```diff
@@ -357,7 +357,7 @@ def request_login_email(publisher:)
     # verify brave gets an internal email copy of confirmation email
     email = ActionMailer::Base.deliveries.find do |message|
       message.to == Rails.application.secrets[:internal_email]
-      message.subject == "<Internal> #{I18n.t('publisher_mailer.confirm_email_change.subject', publication_title: '')}"
```

Comment:
minor: unrelated change? tests seem ok though

Reply:
This test actually started to fail. I suspect it was because we removed the brackets from the email templates.  This seemed to fix it though.

Reply:
hm, I'm having to change this back to having a space in a current branch 🤔